pom.xml

```diff
@@ -1468,9 +1486,14 @@
                     </configuration>
                     <dependencies>
                         <dependency>
-                            <groupId>org.apache.maven.surefire</groupId>
-                            <artifactId>surefire-junit47</artifactId>
-                            <version>${version.surefire.plugin}</version>
+                            <groupId>org.junit.jupiter</groupId>
```

Comment:
I am wondering if we add these dependencies in the parent pom to the dependency section as well as the dependency management sections, then do we need to add these dependencies to every pom?

Reply:
I had tried that initially but the build fails locally saying that a version is required. Once that is resolved by adding the version here in the dependencies section the build fails later on with a failure to pull junit-bom from the packages repo. Something is amiss here because you should be able to declare `junit-bom` in the dependencies section with just the `groupId` and `artifactId`

Reply:
Plugin dependencies can't be managed, unfortunately.  https://issues.apache.org/jira/browse/MNG-2496